### PR TITLE
CART-954 ofi: Update ofi to 1.12.0

### PR DESCRIPTION
**Pull request by frostedcmos**
- Update OFI to release 1.12.0
- Compile ofi without gdrcopy library linkage as we do not require it

PR-repos: libfabric@PR-36

Signed-off-by: Alexander Oganezov <alexander.a.oganezov@intel.com>

### CODE REVIEW

**Review by brianjmurrell — DISMISSED** [review]

**Review by sylviachanoiyee — DISMISSED** [review]

**Review by brianjmurrell — DISMISSED** [review]

**Review by jolivier23 — APPROVED** [review]

File: debian/changelog [hunk 1]
```diff
@@ -2,7 +2,7 @@ daos (1.3.0-3) unstable; urgency=medium
   [ Alexander Oganezov ]
   * Update to ofi v1.12.0 release
 
- -- Alexander Oganezov <alexander.a.oganezov@intel.com> Wed, 10 Mar 2021 05:00:00 -0000
+ -- Alexander Oganezov <alexander.a.oganezov@intel.com> Wed, 10 Mar 2021 05:00:01 -0000
```
Comment (brianjmurrell) [inline debian/changelog:5]: Why was this necessary?  In any case, if you are going to update a timestamp, making it more current would be useful.

**Review by brianjmurrell — APPROVED** [review]

**Review by jolivier23 — DISMISSED** [review]

File: debian/changelog (hunk 1 above)
Comment (frostedcmos) [reply debian/changelog:5]: this was just to retrigger build

Comment (brianjmurrell) [reply debian/changelog:5]: I wondered if that was it.  `git commit --allow-empty` also works for that.  Or you can just do a _Build with Parameters_ right on the classic view page for the PR in Jenkins.

**Comment by frostedcmos** [comment]
This PR is waiting for Maureen to run soak test over weekend with latest 1.12 rpms to verify no new regressions

**Comment by jolivier23** [comment]
Aborted due to race lost on spec file and the CI slowness.  Please rebase

File: utils/rpms/daos.spec [hunk 2]
```diff
@@ -4,10 +4,11 @@
 
 %global mercury_version 2.0.1~rc1-1%{?dist}
 %global libfabric_version 1.12.0
+%global __python %{__python3}
```
Comment (brianjmurrell) [inline utils/rpms/daos.spec:7]: Did something break that requires this?  I'd think the system macros should take care of this.

Comment (frostedcmos) [reply utils/rpms/daos.spec:7]: seems its a result of a bad merge

Comment (frostedcmos) [reply utils/rpms/daos.spec:7]: actually no, it seems to be already in master, no idea why it was added

**Review by jolivier23 — APPROVED** [review]

File: utils/rpms/daos.spec (hunk 2 above)
Comment (brianjmurrell) [reply utils/rpms/daos.spec:7]: Yeah.  I was looking at a subset of commits that included the merge commit which showed this as an added line.

**Review by brianjmurrell — APPROVED** [review]

**Comment by mjean308** [comment]
I ran the soak48 workload over the weekend with daos 1.2 - rpm version:1.1.3-4.5987.g8a2824f3 and libfabric.1.12.0-1
The test ran for 12 continuous hours before it was stopped due to a slurm issue.  In that 12 hours; soak ran 1444 ior jobs that all completed successfully before the slurm issue occurred.
